examples/cloudify-hello-world-example/openstack.yaml

```diff
@@ -0,0 +1,238 @@
+tosca_definitions_version: cloudify_dsl_1_3
+
+imports:
+  - http://www.getcloudify.org/spec/cloudify/4.5/types.yaml
+  # For use with a manager use: plugin:cloudify-openstack-plugin
+  - plugin.yaml
+  - install-script.yaml
+
+inputs:
+
+  auth_url:
+    type: string
+    default: { get_secret: keystone_url }
+
+  username:
+    type: string
+    default: { get_secret: keystone_username }
+
+  password:
+    type: string
+    default: { get_secret: keystone_password }
+
+  project_name:
+    type: string
+    default: { get_secret: keystone_tenant_name }
+
+  region_name:
+    type: string
+
+  external_network_id:
+    type: string
+    description: A network that provisions floating IPs.
+
+  image:
+    type: string
+    description: The UUID of an Ubuntu Trusty 14.04 image in your Openstack account.
+
+  flavor:
+    type: string
+    description: The UUID of small flavor in your Openstack account.
+
+  network_name:
+    type: string
+    description: A name for your new Openstack Network.
+    default: cloudify-hello-world-network
+
+  subnet_name:
+    type: string
+    description: A name for your new Openstack Subnet.
+    default: cloudify-hello-world-subnet
+
+  name_prefix:
+    type: string
+    default: hellow_world
+
+
+dsl_definitions:
+
+  client_config: &client_config
+    auth_url: { get_input: auth_url }
+    username: { get_input: username }
+    password: { get_input: password }
+    project_name: { get_input: project_name }
+    region_name: { get_input: region_name }
+
+node_templates:
+
+  external-network:
+    type: cloudify.nodes.openstack.Network
+    properties:
+      client_config: *client_config
+      use_external_resource: true
+      resource_config:
+        id: { get_input: external_network_id }
+        kwargs:
+          routing:
+            external: true
```

Comment:
@EarthmanT I do not think so we should add `kwargs` since it uses `use_external_resource`

```
  external-network:
    type: cloudify.nodes.openstack.Network
    properties:
      client_config: *client_config
      use_external_resource: true
      resource_config:
        id: { get_input: external_network_id }
```

Reply:
why?

Reply:
because it will be useless since it use `use_external_resource`